tap_gitlab/schemas/commits.json

```diff
@@ -1,39 +1,39 @@
 {
-    "type": "object",
-    "properties": {
-        "id": {
-            "type": "string"
-        },
-        "project_id": {
-            "type": "integer"
-        },
-        "short_id": {
-            "type": "string"
-        },
-        "title": {
-            "type": ["null", "string"]
-        },
-        "author_name": {
-            "type": ["null", "string"]
-        },
-        "author_email": {
-            "type": ["null", "string"]
-        },
-        "committer_name": {
-            "type": ["null", "string"]
-        },
-        "committer_email": {
-            "type": ["null", "string"]
-        },
-        "created_at": {
-            "type": "string",
-            "format": "date-time"
-        },
-        "message": {
-            "type": ["null", "string"]
-        },
-        "allow_failure": {
-            "type": ["null", "boolean"]
-        }
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "project_id": {
+      "type": "integer"
+    },
+    "short_id": {
+      "type": "string"
+    },
+    "title": {
+      "type": ["null", "string"]
+    },
+    "author_name": {
+      "type": ["null", "string"]
+    },
+    "author_email": {
+      "type": ["null", "string"]
+    },
+    "committer_name": {
+      "type": ["null", "string"]
+    },
+    "committer_email": {
+      "type": ["null", "string"]
+    },
+    "created_at": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "message": {
+      "type": ["null", "string"]
+    },
+    "allow_failure": {
+      "type": ["null", "boolean"]
     }
+  }
```

Comment:
Invalid indentation, you may just need to restore changes sicne I don't see any relavent changes in it.